include/phasar/PhasarLLVM/TaintConfig/TaintConfigData.h

```diff
@@ -0,0 +1,54 @@
+/******************************************************************************
+ * Copyright (c) 2023 Fabian Schiebel.
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of LICENSE.txt.
+ *
+ * Contributors:
+ *     Maximilian Leo Huber and others
+ *****************************************************************************/
+
+#ifndef PHASAR_PHASARLLVM_TAINTCONFIG_TAINTCONFIGDATA_H
+#define PHASAR_PHASARLLVM_TAINTCONFIG_TAINTCONFIGDATA_H
+
+#include <string>
+#include <vector>
+
+namespace psr {
+struct TaintConfigData;
+class LLVMProjectIRDB;
+
+struct FunctionData {
+  FunctionData() = default;
+
+  std::string Name;
+  std::string ReturnType;
+  std::vector<int> SourceValues;
+  std::vector<int> SinkValues;
+  std::vector<std::string> SinkStringValues;
+  std::vector<int> SanitizerValues;
+};
+
+struct VariableData {
+  VariableData() = default;
+
+  size_t Line{};
+  std::string Name;
+  std::string Scope;
+  std::string Cat;
+};
+
+struct TaintConfigData {
+  TaintConfigData() = default;
+  explicit TaintConfigData(const std::string &Filepath);
```

Comment:
You probably don't need a ctor here. You can initialize the TaintConfigData from within `parseTaintConfigOrNull()` (maybe separate loading the file from initializing the structs to thrie own functions)